tests/unit/server/bundle_manager/__init__.py

```diff
@@ -60,7 +60,7 @@
 }
 
 
-class BaseBundleManagerTest(unittest.TestCase):
+class TestBase:
```

Comment:
This class has effectively become a "test base class" generic test base that's used for non-BundleManager tests, so it should eventually be moved to a separate file. However, I'm holding off on doing that until #2795 is merged, in order to minimize merge issues / conflicts.